invenio_app_ils/ill/schemas/ill_borrowing_requests/borrowing_request-v1.0.0.json

```diff
@@ -186,7 +186,7 @@
       "title": "Order total in the main currency"
     },
     "type": {
-      "title": "The type of the item requested, normally physical or electronic",
+      "title": "The type of the item requested, normally physical or electronic (Vocabulary)",
```

Comment:
nice, we should do it everywhere

Reply:
It is done extensively in the ACQ and ILL, we can apply it to the rest.